src/api/ha_civo/main.go

```diff
@@ -173,39 +172,40 @@ func CreateVM(name, region string) {
 			fmt.Println("~~~ ", name)
 			password = getInstance.InitialPassword
 			fmt.Println("Password", password)
-      ip = getInstance.PublicIP
-      fmt.Println("Public IP", ip)
+			ip = getInstance.PublicIP
+			fmt.Println("Public IP", ip)
 			fmt.Println()
 			fmt.Println()
 			break
 		}
 		fmt.Println(getInstance.Status)
 		time.Sleep(15 * time.Second)
-  }
-
-  hostKeyCallback, err := knownhosts.New(fmt.Sprintf("%s/.ssh/known_hosts", payload.GetUserName()))
-  if err != nil {
-    panic(err)
-  }
-
-
-  config := &ssh.ClientConfig{
-    User: "root",
-    Auth: []ssh.AuthMethod{
-      ssh.Password(password),
-    },
-    HostKeyCallback: hostKeyCallback,
-  }
-  
-  // connect ot ssh server
-  conn, err := ssh.Dial("tcp", ip, config)
-  if err != nil {
-    panic(err)
-  }
-  defer conn.Close()
-
-
-  var postInstall = `
+	}
+
+	time.Sleep(15 * time.Second)
+	hostKeyCallback, err := knownhosts.New(fmt.Sprintf("%s/.ssh/known_hosts", payload.GetUserName()))
+	if err != nil {
+		fmt.Println(err.Error())
+		return
+	}
+
+	config := &ssh.ClientConfig{
+		User: "ubuntu",
+		Auth: []ssh.AuthMethod{
+			ssh.Password(password),
+		},
+		HostKeyCallback: hostKeyCallback,
+	}
+
+	// connect ot ssh server
+	conn, err := ssh.Dial("tcp", ip+":22", config)
+	if err != nil {
+		fmt.Println(err.Error())
+		return
+	}
```

Comment:
@AvineshTripathi there are some issue with this line
![Thursday 24 November 2022 10:55:56 AM IST](https://user-images.githubusercontent.com/65275144/203795155-5eb7cbea-fda5-4d45-bd0f-d91189cb85eb.png)